### PR TITLE
DMR: Configured repo to have consistent line endings

### PR DESCRIPTION
**Pull request by decodingahmed**
Related issue: [Cross: Git line-endings in all repositories](https://github.com/buerokratt/Cross-functional-requirements/issues/20)

Configured repo to have consistent line endings and friendly for multi-OS development teams.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-All docs live here...
\ No newline at end of file
+All docs live here...
+
+New line
```
Comment (tonistuulNG) [inline docs/README.md]: These new lines are probably a bit extra :)

Comment (guypritchard) [reply docs/README.md]: Aye - if you can address then commit.  I'll put the approval on here so you can get this in when you're ready.

**Review by guypritchard — APPROVED** [review]
Minor comment then go for it!

File: docs/README.md (hunk 1 above)
Comment (decodingahmed) [reply docs/README.md]: Addressed, thank you chaps! :)

**Pull request merged**